include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
+public:
+	enum loglevel
+	{
+		logCRITICAL, logERROR, logWARNING, logDEBUG, logINFO,
```

Comment:
use following name convention for enum members: ll_error. ll is first letters of enum name(log_level). use this as prefix pls. e.g. "ll_error"